freight/api/task_index.py

```diff
@@ -98,6 +78,7 @@ def get(self):
     post_parser = reqparse.RequestParser()
     post_parser.add_argument('app', required=True)
     post_parser.add_argument('user', required=True)
+    post_parser.add_argument('name', default=TaskName.deploy)
```

Comment:
Adding the ability to provide more params is great, I'm also in favor of keeping the name attribut which is used in my case to call multiple fabric tasks (deploy, collectstatic, etc.) with the same sha attribute.